crates/hyperswitch_domain_models/src/payments/payment_attempt.rs

```diff
@@ -1744,13 +1753,39 @@ impl behaviour::Conversion for PaymentAttempt {
     where
         Self: Sized,
     {
-        use crate::type_encryption;
-
         async {
             let connector_payment_id = storage_model
                 .get_optional_connector_transaction_id()
                 .cloned();
 
+            let decrypted_data = crypto_operation(
+                state,
+                common_utils::type_name!(Self::DstType),
+                CryptoOperation::BatchDecrypt(EncryptedPaymentAttempt::to_encryptable(
+                    EncryptedPaymentAttempt {
+                        payment_method_billing_address: storage_model
+                            .payment_method_billing_address,
+                    },
```

Comment:
Why do we need batch encryption for a single field tho?

Reply:
For now there is only one field, later there might be other fields as well which can be added over here